static/app/components/organizations/pageFilters/container.tsx

```diff
@@ -41,10 +41,6 @@ type Props = InitializeUrlStateProps & {
    * filtering contexts like in Dashboard Details.
    */
   disablePersistence?: boolean;
-  /**
-   * Whether to hide the desynced filter alert.
-   */
-  hideDesyncAlert?: boolean;
```

Comment:
This is no longer necessary, we can just check `organization.features.includes('new-page-filter')` instead